specification/security/resource-manager/Microsoft.Security/stable/2021-01-01/alerts.json

```diff
@@ -559,8 +559,8 @@
           }
         ],
         "responses": {
-          "204": {
-            "description": "No Content"
+          "202": {
```

Comment:
this needs to be marked x-ms-long-running-operation if it returns 202

Reply:
Added :)